docs/modules/ROOT/pages/static-site-generators.adoc

```diff
@@ -0,0 +1,94 @@
+= Static Site Generators
+
+Asciidoctor fundamentally converts one source document into one output document.
+To convert more than one document, you need some amount of infrastructure.
+Simple situations may find it convenient to use built-in Asciidoctor capabilities or scripts, but are unlikely to be appropriate for any but the smallest sites.
+Static site generators organize the conversion of multiple source documents into a functioning static web site.
+
+== What does a Static Site Generator (ssg) do?
+
+A static site generator (ssg) gathers sources from one or more locations, transforms them, and writes the results into a file system, concrete or virtual, ready for deployment to a web server.
```

Comment:
I think that's too much detail for one sentence, maybe:

"creates (or generates) a set of static assets which can be served by a web server)".

I feel like the "virtual/concrete file system" part is too specific/technical.

Reply:
yes....
`A static site generator gathers sources from one or more locations, transforms them, and organizes the resulting static assets into a complete site ready for deployment directly to a web server.`?

Reply:
Yes that's better!
Still a bit convoluted but let's hear what others think :wink: